README.md

```diff
@@ -708,3 +855,57 @@ to find the bundled integration test input file for the given language.
 
 For example, the playground can be run in a tmux pane, with your editor
 of choice open in another.
+
+## Related Tools
+
+### Tree-Sitter Specific
+
+* [Syntax Tree Playground][tree-sitter-playground]: An interactive
+  playground for experimenting with Tree-sitter and its query language.
+
+### Meta and Multi-Language Formatters
+
+* [format-all]: A formatter orchestrator for Emacs.
+* [null-ls.nvim]: An LSP framework for Neovim that facilitates formatter
+  orchestration.
+* [prettier]: A formatter with support for multiple (web-development
+  related) languages.
+* [treefmt]: A general formatter orchestrator, which unifies formatters
+  under a common interface.
+
+### Related Formatters
+
+* [gofmt]: A (_the_) formatter for Go.
```

Comment:
And major inspiration for the style of our formatters.

Reply:
Added in cd20b49a278a